airbyte-integrations/connectors/source-close-com/acceptance-test-config.yml

```diff
@@ -14,10 +14,24 @@ tests:
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: ["meeting_activities", "missed_call_tasks", "email_followup_tasks", "answered_detached_call_tasks",
-                      "incoming_sms_tasks", "zoom_connected_accounts", "custom_email_connected_accounts",
-                      "google_connected_accounts", "send_as", "email_sequences", "dialer", "email_bulk_actions",
-                      "sequence_subscription_bulk_actions", "delete_bulk_actions"]
+      empty_streams:
```

Comment:
there should be no empty streams if we're going to certify this connector to beta

Reply:
This is not a certification to beta

Reply:
I was trying to avoid empty streams but didn't succeed. We cannot to populate some tasks from our side and `send_as` requires payment.  It also makes impossible for now to migrate SAT config to new version